### ISSUE TITLE
Negative Cl in FlightData GUI

### ISSUE THREAD

**Issue by DaMichel** [ferram4/Ferram-Aerospace-Research#57]
I made a dual body aircraft for which the flight data gui shows a negative cl.
https://www.dropbox.com/s/iu4jloqi8b6aowz/dualbodybug.jpg?dl=0
https://www.dropbox.com/s/g519pevg3z50c54/Dual%20Body%20Glider.craft?dl=0
Accordingly, the shown L/D ratio is -1.42.

Judging by the glide path, i estimate that the plane actually has a L/D above 10.

**Comment by ferram4** [comment]
This appears to be fixed within the voxelAeroPort branch, and so I'll be marking this as closed once that is merged into master.

**Comment by ferram4** [comment]
Fixed with voxel aero update.
